homeassistant/components/splunk/manifest.json

```diff
@@ -5,5 +5,5 @@
   "documentation": "https://www.home-assistant.io/integrations/splunk",
   "iot_class": "local_push",
   "loggers": ["hass_splunk"],
-  "requirements": ["hass_splunk==0.1.1"]
+  "requirements": ["hass_splunk==0.1.2"]
```

Comment:
Please tag a github release for the commit corresponding to the PyPI release, to make it easier for the community to track changes. Thanks!

Reply:
Can you also tag 0.1.1 and adjust the diff link?